apps/electron/src/helper/db/base-db-adapter.ts

```diff
@@ -79,21 +79,34 @@ export abstract class BaseSQLiteAdapter {
     }
   }
 
-  async getUpdates() {
+  async getUpdates(docId?: string) {
```

Comment:
why doc id could be `undefined`?

Reply:
for convention rows without doc_id is considered the root doc's update

Reply:
i think this may not be a good design and make the implementation complicated. i can change it to always populate the db in next canary version when we consider migration, is this ok?